aggregators/internal/hdrhistogram/hdrhistogram.go

```diff
@@ -72,13 +73,28 @@ type HistogramRepresentation struct {
 	CountsRep             HybridCountsRep
 }
 
+var histogramPool = sync.Pool{
+	New: func() interface{} {
+		return &HistogramRepresentation{
+			LowestTrackableValue:  lowestTrackableValue,
+			HighestTrackableValue: highestTrackableValue,
+			SignificantFigures:    significantFigures,
+		}
+	},
+}
+
```

Comment:
After the latest round of improvements in histogram representation, can you rerun the benchmarks and see if we really require to pool histograms.

Reply:
Will do